spyder/utils/icon_manager.py

```diff
@@ -401,7 +421,7 @@ def get_std_icon(self, name, size=None):
         else:
             return QIcon(icon.pixmap(size, size))
 
-    def get_icon(self, name, resample=False):
+    def get_icon(self, name, resample=True):
```

Comment:
Please revert this change to be `resample=False` because it's only needed in one place as `True`:

https://github.com/spyder-ide/spyder/blob/aeacc49881eb58abdf6e102a7f3eef847103f4b2/spyder/app/mainwindow.py#L177-L179